frontend/src/components/CytoscapeGraph/MiniGraphCard.tsx

```diff
@@ -177,7 +181,7 @@ class MiniGraphCard extends React.Component<MiniGraphCardProps, MiniGraphCardSta
           </CardBody>
         </Card>
         <TimeDurationModal
-          customDuration={false}
+          customDuration={true}
```

Comment:
Passing `custionDuration={false}` seems to be right. The indicator is what seems to have the wrong argument. I tried to explain this in your previous (now closed) PR, but probably my review comment was somewhat vague.